apcd-cms/src/apps/admin_extension/views.py

```diff
@@ -102,127 +82,91 @@ def getDate(row):
-        context['extensions'] = []
-
-        try:
-            page_num = int(self.request.GET.get('page'))
-        except:
-            page_num = 1
-
-        def getDate(row):
-            date = row[9]
-            return date if date is not None else parser.parse('1-1-0001')
-
-        extension_content = sorted(extension_content, key=lambda row:getDate(row), reverse=True)  # sort extensions by newest to oldest
-
-        extension_table_entries = []       
-        for extension in extension_content:
-            # to be used by paginator
-            extension_table_entries.append(_set_extensions(extension))
-            # to be able to access any extension in a template
-            context['extensions'].append(_set_extensions(extension))
-            entity_name = title_case(extension[18])
-            status = title_case(extension[7])
-            outcome = title_case(extension[8])
-            if entity_name not in context['org_options']:
-                context['org_options'].append(entity_name)
-                context['org_options'] = sorted(context['org_options'], key=lambda x: (x != 'All', x))
-                # Remove empty strings
-                context['org_options'] = [option for option in context['org_options'] if option != '']
-            if status not in context['status_options']:
-                context['status_options'].append(status)
-                context['status_options'] = sorted(context['status_options'], key=lambda x: (x != 'All', x))
-            if outcome not in context['outcome_options']:
-                context['outcome_options'].append(outcome)
-                context['outcome_options'] = sorted(context['outcome_options'], key=lambda x: (x is not None, x))
-
-        queryStr = ''
-        status_filter = self.request.GET.get('status')
-        org_filter = self.request.GET.get('org')
-
-        context['selected_status'] = None
-        if status_filter is not None and status_filter != 'All':
-            context['selected_status'] = status_filter
-            queryStr += f'&status={status_filter}'
-            extension_table_entries = table_filter(status_filter, extension_table_entries, 'status')
-
-        context['selected_org'] = None
-        if org_filter is not None and org_filter != 'All':
-            context['selected_org'] = org_filter
-            queryStr += f'&org={org_filter}'
-            extension_table_entries = table_filter(org_filter.replace("(", "").replace(")",""), extension_table_entries, 'entity_name')
-
-        context['query_str'] = queryStr
-        context.update(paginator(self.request, extension_table_entries))
-        context['pagination_url_namespaces'] = 'admin_extension:list_extensions'
-
-        return context
-
 # function converts int value in the format YYYYMM to a string with abbreviated month and year
 def _get_applicable_data_period(value):
     try:
         return datetime.strptime(str(value), '%Y%m').strftime('%b. %Y')
```

Comment:
I'm getting an error locally on datetime in `return datetime.strptime(str(value), '%Y%m').strftime('%b. %Y')` that datetime is not defined. Possibly could use dateUtil Chandra made in this PR?

[https://github.com/TACC/Core-CMS-Custom/pull/338](https://github.com/TACC/Core-CMS-Custom/pull/338)